app/src/main/java/io/appium/uiautomator2/utils/Attribute.java

```diff
@@ -45,6 +45,24 @@ public enum Attribute {
     TEXT(new String[]{"text", "name"}),
     HINT(new String[]{"hint"}),
     EXTRAS(new String[]{"extras"}),
+    IMPORTANT_FOR_ACCESSIBILITY(new String[]{"importantForAccessibility"}),
+    SCREEN_READER_FOCUSABLE(new String[]{"screenReaderFocusable"}),
+    INPUT_TYPE(new String[]{"inputType"}),
+    DRAWING_ORDER(new String[]{"drawingOrder"}),
+    SHOWING_HINT_TEXT(new String[]{"showingHintText"}),
+    TEXT_ENTRY_KEY(new String[]{"textEntryKey"}),
+    MULTI_LINE(new String[]{"multiLine"}),
+    DISMISSABLE(new String[]{"dismissable"}),
+    ACCESSIBILITY_FOCUSED(new String[]{"accessibilityFocused"}),
+    HEADING(new String[]{"heading"}),
+    LIVE_REGION(new String[]{"liveRegion"}),
+    CONTEXT_CLICKABLE(new String[]{"contextClickable"}),
+    MAX_TEXT_LENGTH(new String[]{"maxTextLength"}),
+    CONTENT_INVALID(new String[]{"contentInvalid"}),
+    ERROR_TEXT(new String[]{"errorText", "error"}),
+    PANE_TITLE(new String[]{"paneTitle"}),
+    ACTIONS(new String[]{"actions"}),
+
```

Comment:
it makes sense to define new items after CONTENT_SIZE to preserve the ordinality